lib/main.dart

```diff
@@ -147,6 +150,16 @@ class _OnyxAppState extends State<OnyxApp> {
   }
 }
 
+
+class MyCustomScrollBehavior extends MaterialScrollBehavior {
+  @override
+  Set<PointerDeviceKind> get dragDevices => {
+    PointerDeviceKind.touch,
+    PointerDeviceKind.mouse,
+  };
+}
+
+
```

Comment:
so this is not part of #61 (text scaling) but for #62 (top bar)?
 – we should have a separate branch and pull request for #62 ! It is important to keep one branch (new branch, based on master) and one pull request for each issue, otherwise it is confusing to review and merge.